.circleci/config.yml

```diff
@@ -2,39 +2,44 @@ version: 2.1
 orbs:
   codecov: codecov/codecov@3.2.2
   node: circleci/node@5.0.2
+  browser-tools: circleci/browser-tools@1.3.0
```

Comment:
This is how you install the chrome driver on the new images

[Docs](https://circleci.com/developer/images/image/cimg/python?utm_source=google&utm_medium=sem&utm_campaign=sem-google-dg--uscan-en-dsa-maxConv-auth-nb&utm_term=g_-_c__dsa_&utm_content=&gclid=Cj0KCQjwidSWBhDdARIsAIoTVb3fSSKCqxNZow2KGagY8Cr-5KfmBfB6zDHJWCmDme7j4i69IOHp__waAiy5EALw_wcB#browsers)